pkg/driver/driver.go

```diff
@@ -124,3 +127,23 @@ func (d *Driver) Run() error {
 	klog.Infof("Listening for connections on address: %#v", listener.Addr())
 	return d.srv.Serve(listener)
 }
+
+func parseTagsFromStr(tagStr string) map[string]string {
+	defer func() {
+		if r := recover(); r != nil {
+			klog.Errorf("Failed to parse input tag string: %v", tagStr)
```

Comment:
I can add a test here.

The panicking and recovering is when there is a typo in the input or the key value pair is not of the form `key`:`value`. For example: `key-value key;value`

Reply:
Also WDYT about parsing at startup and fast failing instead of at createvolume time?

With current implementation, driver will startup and mount/unmount will still work but then all createvolumes will fail and the user will have to do debugging. Whereas if we fast fail they will presumably see the error message printed by the plugin